site/_includes/layouts/chrome-100.njk

```diff
@@ -0,0 +1,34 @@
+{% extends "layouts/base.njk" %}
+
+{% block chromescripts %}
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/3.9.1/gsap.min.js"></script>
```

Comment:
Note to @devnook: we'll have to edit https://github.com/GoogleChrome/developer.chrome.com/blob/48f1445bab01a3fd39cea7b4c0f896c7159c118d/server/index.js#L50 in a follow-up PR to prevent warnings from being reported.

Reply:
Done

Reply:
Done!

Reply:
we cannot launch with CSP warnings, as this will hit our lighthouse score as well as make scripts non executing (i think?)

Reply:
Our CSP policy looks is in report-only mode, but yeah, since we know what origins will be used in advance, we should get them whitelisted.

(I'm assuming there will be a period of time in which we check in this PR before we have officially "launched".)

Reply:
For the record: I'm not at all hyped about whitelisting whole cloudflare cdn. But we can try to find a more granular solution in a next step to not block this Pr.